LambdaEngine/Source/Resources/ResourceManager.cpp

```diff
@@ -110,7 +116,8 @@ namespace LambdaEngine
 		SAFEDELETE_ALL(s_Meshes);
 		SAFEDELETE_ALL(s_Materials);
 		SAFEDELETE_ALL(s_Animations);
-		SAFEDELETE_ALL(s_SoundEffects);
+		SAFEDELETE_ALL(s_SoundEffects3D);
+		SAFEDELETE_ALL(s_SoundEffects2D);
 
```

Comment:
I might be way off here but shouldn't there be a SAFEDELETE_ALL(s_Music) as well?

Reply:
Bra